spec/spec.md

```diff
@@ -598,7 +598,7 @@ where:
 
 - `payloads`: one or more base64-encoded transaction payload to sign
 - `options`: a JSON object, containing:
-  - `commitment`: (optional) if set, one of `processed`, `confirmed`, or `finalized`, specifying the desired commitment level that must be reached for this request to be successful. If not set, defaults to `confirmed`.
+  - `min_context_slot`: (optional) if set, the minimum slot number that the request can be evaluated at
```

Comment:
I updated the Solana docs for this parameter, and those edits might be helpful here too.

https://github.com/solana-labs/solana/commit/a0698d4cc3ba88510cdd2b869e9c7bc8448ab86a